globals.yml

```diff
@@ -22,5 +22,3 @@ mongodb_conf_bind_ip: 0.0.0.0
 mongodb_host: "{{ hostvars[groups['mongodb'][0]].internal_ip }}"
 mongodb_port: 27017
 
-admin_user: administrator@gds.tsuru.gov
-admin_password: admin123
```

Comment:
How about just commenting those vars here and also add the ones defined in the secure file ?
The problem I see here is that once the vars secured with vault we have no idea about what are the vars names. 
It makes things hard to debug, not to mention that the project becomes almost impossible to reuse.